Publish docs on readthedocs (#1254)
GeotrekCE/Geotrek-rando-v3: README.md

<h1 align="center">Geotrek-rando</h1>

<p align="center"><img alt="geotrek rando image" src="/docs/img/geotrek-rando.png"></p>

<p align="center">
<a href="https://geotrek-rando-v3.readthedocs.io/" rel="nofollow"><img alt="Documentation" src="https://img.shields.io/badge/Read%20the%20Docs-8CA1AF?logo=readthedocs&logoColor=fff" style="max-width:100%;"></a>

> <a href="https://gtr3demo.ecrins-parcnational.fr/" rel="nofollow"><img alt="Démo gtr3 écrins" src="https://img.shields.io/badge/Demo-PN Ecrins-orange.svg" style="max-width:100%;"></a>
> <a href="https://gtr3demo.ecrins-parcnational.fr/" rel="nofollow"><img alt="Démo gtr3 geotrek.fr" src="https://img.shields.io/badge/Demo-Geotrek.svg" style="max-width:100%;"></a>

Reviewer: ```suggestion
<a href="https://gtr3demo.ecrins-parcnational.fr/" rel="nofollow"><img alt="Geotrek Rando demo by Ecrins National Park" src="https://img.shields.io/badge/Demo-PN Ecrins-orange.svg" style="max-width:100%;"></a>
<a href="https://demo-rando.geotrek.fr/" rel="nofollow"><img alt="Geotrek Rando demo" src="https://img.shields.io/badge/Demo-Geotrek.svg" style="max-width:100%;"></a>
```

<a href="https://github.com/GeotrekCE/Geotrek-rando-v3/actions/workflows/e2e.yml" rel="nofollow"><img alt="Status Test E2E" src="https://github.com/geotrekce/Geotrek-rando-v3/actions/workflows/release.yml/badge.svg" style="max-width:100%;"></a>
</p>

<p align="center">
    <a href="#website"><b>Website</b></a>  •  
    <a href="#features"><b>Features</b></a>  •  
    <a href="#admin-documentation"><b>Admin documentation</b></a>  •  
    <a href="#developer-documentation"><b>Developer documentation</b></a>  •  
</p>
<p align="center">
    <a href="#architecture-decision-record"><b>Architecture decision record </b></a>  •  
    <a href="#support"><b>Support</b></a>  •   
    <a href="#contribution"><b>Contribution</b></a>  •  
    <a href="#thanks-to-all-contributors-"><b>Contributors</b></a>  •  
    <a href="#license"><b>License</b></a>  • 
</p>

![Search](/docs/img/home_ecrins.png)

## Website

Geotrek-rando is the public web application displaying the interface you can use to value your territory treks and tourism products! 

> You can find two demonstration website at the following addresses : 

Reviewer: ```suggestion
You can find two demonstration websites at the following links:
```

<h1 align="center">Geotrek-rando</h1>

<p align="center"><img alt="geotrek rando image" src="https://res.cloudinary.com/do0y4qr5g/image/upload/v1730274422/Geotrek/geotrek-rando_o0dx7o.png"></p>

> <p align="center">
> <a href="https://github.com/GeotrekCE/Geotrek-rando-v3/releases/latest" rel="nofollow"><img alt="Release" src="https://img.shields.io/github/release/GeotrekCE/Geotrek-rando-v3.svg" style="max-width:100%;"></a>
> <a href="https://geotrek-rando-v3.readthedocs.io/" rel="nofollow"><img alt="Doc material for mkdocs" src="https://img.shields.io/badge/Doc-material for mkdocs-purple.svg" style="max-width:100%;"></a>
> <a href="https://gtr3demo.ecrins-parcnational.fr/" rel="nofollow"><img alt="Démo gtr3" src="https://img.shields.io/badge/GTR3-demo-orange.svg" style="max-width:100%;"></a>
> <a href="https://matrix.to/#/%23geotrek:matrix.org" rel="nofollow"><img alt="Chat matrix" src="https://img.shields.io/badge/Chat-matrix-green.svg" style="max-width:100%;"></a>
> <a href="https://groups.google.com/g/geotrek-fr" rel="nofollow"><img alt="Google group" src="https://img.shields.io/badge/Google-group-red.svg" style="max-width:100%;"></a>
> </p>

Reviewer 2: ```suggestion
<p align="center">
<a href="https://geotrek-rando-v3.readthedocs.io/" rel="nofollow"><img alt="Documentation" src="https://img.shields.io/badge/Read%20the%20Docs-8CA1AF?logo=readthedocs&logoColor=fff" style="max-width:100%;"></a>
<a href="https://gtr3demo.ecrins-parcnational.fr/" rel="nofollow"><img alt="Démo gtr3 écrins" src="https://img.shields.io/badge/Demo-PN Ecrins-orange.svg" style="max-width:100%;"></a>
<a href="https://gtr3demo.ecrins-parcnational.fr/" rel="nofollow"><img alt="Démo gtr3 geotrek.fr" src="https://img.shields.io/badge/Demo-Geotrek.svg" style="max-width:100%;"></a>
<a href="https://matrix.to/#/%23geotrek:matrix.org" rel="nofollow"><img alt="Chat matrix" src="https://img.shields.io/badge/Matrix-000?logo=matrix&logoColor=fff" style="max-width:100%;"></a>
<a href="https://groups.google.com/g/geotrek-fr" rel="nofollow"><img alt="Google group" src="https://img.shields.io/badge/Google-group-red.svg" style="max-width:100%;"></a>
</p>

<p align="center">
<a href="https://github.com/GeotrekCE/Geotrek-rando-v3/releases/latest" rel="nofollow"><img alt="Release" src="https://img.shields.io/github/release/GeotrekCE/Geotrek-rando-v3.svg" style="max-width:100%;"></a>
<a href="https://github.com/GeotrekCE/Geotrek-rando-v3/actions/workflows/e2e.yml" rel="nofollow"><img alt="Status Test E2E" src="https://github.com/geotrekce/Geotrek-rando-v3/actions/workflows/e2e.yml/badge.svg" style="max-width:100%;"></a>
<a href="https://github.com/GeotrekCE/Geotrek-rando-v3/actions/workflows/e2e.yml" rel="nofollow"><img alt="Status Test E2E" src="https://github.com/geotrekce/Geotrek-rando-v3/actions/workflows/action-intall-and-test.yml/badge.svg" style="max-width:100%;"></a>
<a href="https://github.com/GeotrekCE/Geotrek-rando-v3/actions/workflows/e2e.yml" rel="nofollow"><img alt="Status Test E2E" src="https://github.com/geotrekce/Geotrek-rando-v3/actions/workflows/release.yml/badge.svg" style="max-width:100%;"></a>
</p>
```

<p align="center">
    <a href="#website"><b>Website</b></a>  •  
    <a href="#features"><b>Features</b></a>  •  
    <a href="#admin-documentation"><b>Admin documentation</b></a>  •  
    <a href="#developer-documentation"><b>Developer documentation</b></a>  •  
    <a href="#getting-started"><b>Getting started</b></a>
</p>
<p align="center">
    <a href="#architecture-decision-record"><b>Architecture decision record </b></a>  •  
    <a href="#support"><b>Support</b></a>  •   
    <a href="#contribution"><b>Contribution</b></a>  •  
    <a href="#thanks-to-all-contributors-"><b>Contributors</b></a>  •  
    <a href="#license"><b>License</b></a>  • 
</p>

![Search](/docs/img/home_ecrins.png)

## Website

Geotrek-rando is the public web application displaying the interface you can use to value your territory treks and tourism products! 

> Demo available at [https://gtr3demo.ecrins-parcnational.fr](https://gtr3demo.ecrins-parcnational.fr).

Author: On pourrait aussi mettre directement 2 ou 3 exemples de vrais portails pour montrer des vrais usages, moins basiques, non ?
Mettre les liens des admins, etc... pourquoi pas, mais faut pas que ça devienne un peu complexe de s'y retrouver dans trop de liens et d'infos. Pas d'avis tranché donc.

> Demo available at [https://gtr3demo.ecrins-parcnational.fr](https://gtr3demo.ecrins-parcnational.fr).

Reviewer 3: J'ai créé une section "Examples of Geotrek-rando portals" avec 4 randos + un montage d'images pour illustrer.

> Demo available at [https://gtr3demo.ecrins-parcnational.fr](https://gtr3demo.ecrins-parcnational.fr).

Reviewer 3: pour les liens de l'admin je n'ai rien fait pour l'instant, pas trop d'idée sur la façon de les présenter dans le readme.

- **Responsive PWA** interface with offline access
- **Customizable** homepage
- **Advanced search** with filters and **interactive map**
- **Detailed information** on each trek, outdoor activities, touristic services and events
- **Download options** (PDF, GPX, KML)
- **Static pages** for general info
- **Interactive mapping** with elevation profiles
- **API integration** with Geotrek-Admin for real-time data
- **Design and content customization** (theme, colors, maps)
- **SEO optimization** with server-side rendering.

## Admin documentation

- [Production setup](https://geotrek-rando-v3.readthedocs.io/latest/installation/)
- [Customization](https://geotrek-rando-v3.readthedocs.io/latest/customization/customization-introduction/)

## Developer documentation

- [Development setup](https://geotrek-rando-v3.readthedocs.io/latest/development/installation/)
- [Deployment](https://geotrek-rando-v3.readthedocs.io/latest/development/deployment/)

## Getting started

1. Follow the [Development setup](https://geotrek-rando-v3.readthedocs.io/latest/development/installation/) instructions

> 2. Launch the app at [localhost:3000](http://localhost:3000)

Reviewer 2: Enelver cette section car redondant avec la section "Develeper documentation" ? Ou à minima la mettre à un niveau de titre inférieure car liée à la documentation de développement.

Reviewer 3: Ok, c'est redondant en effet, je retire cette section.
